### ISSUE TITLE
Telegram group invite links.

### ISSUE THREAD

**Issue by Techwolf** [Aseman-Land/Cutegram#163]
Like to have a way to paste a group invite link to join a group. Example: tg://join?invite=B35mgwIt3-oCn_xxxxxxx that is gotten from https://telegram.me/joinchat/B35mgwIt3-oCn_xxxxxxxx that was posted on a social website.

**Comment by jeifour** [comment]
Yeah I wanted to join some chats that were advertised here http://telegramchats.org/ but I can't figure out how to open them in Cutegram.

**Comment by MichalKarnicki** [comment]
Since Bardia already replied in a newer bug about the same thing, this one should be closed.

See: https://github.com/Aseman-Land/Cutegram/issues/179

**Comment by Rebecca10** [comment]
How can I registrar

**Comment by deepika1401** [comment]
hi

**Comment by juan576** [comment]
alguem sabe quando vai ser as aulas ????

**Comment by princessalao** [comment]
Unsubscribe

**Comment by Ksetiadi** [comment]
Hi

**Comment by Ksetiadi** [comment]
https://t.me/joinchat/607esica_WEwM2Rh

**Comment by Ksetiadi** [comment]
Hi

**Comment by Adryanaje** [comment]
Hi

**Comment by thaisa2** [comment]
Como faço pra entrar neste grupo do telegram  tg:join?invite=XoPcMW37h2YzNTEx

**Comment by sassysmith01** [comment]
Not sure if this is the right place to post this issue...im using vidsley to watch GOT and non of the parts where they speak in another language have subtitles. So i cant understand anything they are saying or understand whats going on please fix this issue

**Comment by forsage555** [comment]
Hii

**Comment by PARAMRR** [comment]
Withdraw problem  i can't able to withdraw my divided

**Comment by 8400099210** [comment]
Great 🤞

**Comment by jose5350** [comment]
Hol

**Comment by HolasoyHola123** [comment]
Hola

**Comment by SBININBB481** [comment]
Hi

**Comment by Hwjweiehrh3ieirhdjwosshbdhr** [comment]
Join Group
tg:join?invite=RQFFIpbQZmhiOTlh

**Comment by Naty8989** [comment]
Vbbb

**Comment by THEBASTER1** [comment]
tg:join?invite=iLX66UwKMOg3OWVi

Hi Om New I take a invitatiom. CAN i redem my bonus?

**Comment by mihirevaiteraitefaaite-cmyk** [comment]
Hii how are you?
